### ISSUE TITLE
How do I configure rinetd.conf to allow multiple IP?

### ISSUE THREAD

**Issue by fivetime** [samhocevar/rinetd#23]
How do I configure rinetd.conf to allow multiple IP, to have one IP address per line like this?

```
allow 14.49.36.1
allow 14.46.200.1
0.0.0.0 24057 127.0.0.1 6120
```

But after testing, I found that this configuration is not working, it is only valid for the last line of allow.

**Comment by samhocevar** [comment]
Thanks, you actually found a bug in the configuration parser! The very first rule of the very first server gets ignored.

I have fixed this in git; in the meantime, you can mitigate this problem by adding a dummy rule that will never be matched, such as:

    deny 255.255.255.255

**Comment by fivetime** [comment]
I see that you have changed the code. Has this bug been fixed?

**Comment by samhocevar** [comment]
Yes, the git repository has the fix. It will be in the next release.
